### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kirkkwang**

### CODE REVIEW

File: app/javascript/components/pages/MedicationList.test.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ import React from "react";
 import Enzyme, { shallow } from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
 import MedicationList from "./MedicationList";
+// import MockMeds from "../MockMeds"
```
Comment (sjproctor) [inline app/javascript/components/pages/MedicationList.test.js:5]: Are you still using line 5?

File: app/javascript/components/pages/PatientInfo.test.js [hunk 2]
```diff
@@ -26,4 +26,23 @@ describe("when PatientInfo renders", () => {
...
+    info.instance().handleModalOpen();
+    expect(info.state().modalOpen).toEqual(false);
+  });
+  it("Test click event", () => {
+    const mockCallBack = jest.fn();
+    const patientInfoWrapper = shallow(<PatientInfo deletePatient={mockCallBack} />);
+    const button = patientInfoWrapper.find('button[children="Delete Patient"]');
+    button.simulate("click");
+    expect(mockCallBack.mock.calls.length).toEqual(1);
+  });
```
Comment (sjproctor) [inline app/javascript/components/pages/PatientInfo.test.js:47]: Nice job on the testing!

**Review by sjproctor — APPROVED** [review]
Good work!!

File: app/javascript/components/MockMeds.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+
+medications = [
```
Comment (archae0pteryx) [inline app/javascript/components/MockMeds.js:2]: do you need to mutate this object? Should it be a const?

File: app/javascript/components/pages/MedicationList.test.js (hunk 1 above)
Comment (archae0pteryx) [inline app/javascript/components/pages/MedicationList.test.js:5]: every time a comment gets merges, an angel loses it's wings.

File: app/javascript/components/pages/PatientInfo.test.js [hunk 4]
```diff
@@ -26,4 +26,23 @@ describe("when PatientInfo renders", () => {
...
+});
```
Comment (archae0pteryx) [inline app/javascript/components/pages/PatientInfo.test.js:48]: nitpick: remember to format your code.

**Review by archae0pteryx — APPROVED** [review]
LGTM... only one angel lost it's wings.

File: app/javascript/components/pages/MedicationList.test.js (hunk 1 above)
Comment (kirkkwang) [reply app/javascript/components/pages/MedicationList.test.js:5]: dammit, swear one day i'll catch them all

**Pull request merged**